addons/hr_holidays/tests/__init__.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import test_holidays_flow
-
+import test_holidays_flow
```

Comment:
I was removing the extra blank line to satisfy the needs of my linter thought I'd change that import to match the mainstream way of doing the import while I was at it.
Want me to go back and leave it the way it was ?